### DISCUSSION TITLE
2022: the block developer’s year in review

### DISCUSSION THREAD

**Discussion by mburridge** [WordPress/developer-blog-content#37] [Topic Ideas]
A post wrapping up 2022 and summarising all the new developments and resources that became available during 2022.

e.g.
- workshops/courses/tutorials on learn.wordpress.org
- wordpress.tv videos
- dev notes
- updates to create-block

Can you think of any other ideas not covered in the above list?

To be published before the end of 2022.

**Comment by mburridge** [comment]
[The post](https://developer.wordpress.org/news/?p=572&preview=1&_ppp=d37dab29c4) has been scheduled to be published on Wed 28 Dec.

**Comment by bph** [comment]
Published: [2022: the block developer’s year in review](https://developer.wordpress.org/news/2022/12/2022-the-block-developers-year-in-review/)
